packages/core/src/types/v3/PaginatedResults.ts

```diff
@@ -0,0 +1,5 @@
+export interface PaginatedResponse {
```

Comment:
Do all paginated results have the same signature?

Maybe we could do something like this then:
```
export interface PaginatedResponse<T> {
    results: T[]
    page: number;
    total_results: number;
    total_pages: number;
}
```

This way we also do not have to repeat ourselves for the "results" field

Reply:
Sorry I  thought I was submitting a pr to my own repo, and I do not have this fully ready yet. I will submit this again later this evening.

Reply:
So far all the /search/ enpoints do appear to have the signature. I am trying to see what else has them at the moment